src/cgame/default/Makefile.am

```diff
@@ -66,5 +66,6 @@ cgame_la_LDFLAGS = \
 cgame_la_LIBADD = \
 	viewcontrollers/libviewcontrollers.la \
 	$(top_builddir)/src/game/default/libpmove.la \
-	$(top_builddir)/src/libswap.la
+	$(top_builddir)/src/libswap.la \
+	$(top_builddir)/src/libmatrix.la
```

Comment:
It may not be required in the end, but an doing my best to keep it up to date with my projects.